packages/web-app/src/containers/transactionModals/DepositModal.tsx

```diff
@@ -23,13 +23,24 @@ const DepositModal: React.FC = () => {
   const { network } = useNetwork();
   const { alert } = useAlertContext();
   const navigate = useNavigate();
-  const amountToDeposit = 100;
+  const [input, setInput] = useState({
+    amount: '100',
+    tokenAddress: '',
+  });
+
+  useEffect(() => {
+    daoDetails?.address && setInput({ ...input, tokenAddress: daoDetails?.address as string });
```

Comment:
we shouldn't depend on daoDetails?.address since this field is not used anymore. Maybe should tokenAddress be used?

Reply:
ok